### DISCUSSION TITLE
Specktrum protocol likely outdated

### DISCUSSION THREAD

**Discussion by nerdCopter** [emuflight/EmuFlight#963] [General]
@SpektrumRC , although EmuFlight is essentially dead, would you be interested in bringing the code-base up to par?
Additionally, Emu also does not support SPI based DSMX/DSM2 at all. The Betaflight PR/merge (https://github.com/betaflight/betaflight/pull/7210) was after the forking.

ref: https://github.com/betaflight/betaflight/pull/13383

**Comment by SpektrumRC** [comment]
Unfortunately its tough to justify the effort, at least on the SRXL2 side. Best bet is if the interest is there, somebody can do a merge from the betaflight implementation and work out the kinks. 

Regarding the SPI support side, that was never actually done/sanctioned by us and came about from reverse engineering efforts. With that in mind, we definitely won't be involved in that one.

**Reply by nerdCopter** [reply]
I appreciate your reply, thank you.
